source/guides/branding.rst

```diff
@@ -0,0 +1,90 @@
+============================================
+Mattermost Branding and Naming Guidelines 
+============================================
+
+The Mattermost branding and naming guidelines are intended to help keep users safe and to support the development of community integrations through a set of consistent guidelines for use of the Mattermost name and brand. 
+
+OVERVIEW: 
+
+.. contents::
+  :backlinks: top
+  :local:
+
+--------------------------------------
+Mattermost Visual Design Guidelines 
+--------------------------------------
+
+Mattermost Logo 
+-------------------------------
+
+For details on Mattermost logo usage guidelines please visit: https://www.mattermost.org/brand-guidelines/
+
+Mattermost User Experience 
+-------------------------------
+
+For details on the visual design of the Mattermost user experience, please see: https://docs.mattermost.com/developer/fx-guidelines.html
+
+-------------------------------
+Naming Usage Guidelines
+-------------------------------
+
+Using the Mattermost name is generally fine so long as it’s used in a non-commercial way and does not imply endorsement by Mattermost, Inc.
+
+Naming Integrations 
+-------------------------------
+
+The following guidelines are intended to help developers of Mattermost integrations offer clear, consistent naming across the broader community. 
+
+- IMPORTANT: Please do not name your integrations or apps starting with, or only using, the name “Mattermost”, as this may confuse end users as to where to find support
```

Comment:
Propose

```- IMPORTANT: Please do not start with or use the name "Mattermost" for your integration as this may confuse  users on where to find support.
```

 - add missing period
 - rephrase to reduce number of commas used in the sentence
 - replace 'end user' with 'user' since the latter captures all users of the integration